src/recurrenceIterator.ts

```diff
@@ -1,8 +1,9 @@
 // Takes in a collection of sequences.
 // Returns a series of sequences in the order in which they'll run.
 import { RRule } from 'rrule-alt';
-import moment from 'moment';
+import * as moment from 'moment-timezone';
```

Comment:
`moment-timezone` is an extension of `moment` with correct typings so we use that instead